### ISSUE TITLE
No definition for BCrypt.HashPassword

### ISSUE THREAD

**Issue by Itay080** [BcryptNet/bcrypt.net#8]
I have used the package manager console to get the latest release, yet BCrypt.HasPassword is not defined thus not useable.

**Comment by ChrisMcKee** [comment]
You mean HashPassword right?

```
using BCrypt.Net;
...
var hash = BCrypt.HashPassword("string");
```

**Comment by Itay080** [comment]
Whoops.
Pardon me, I used the namespace instead of the class, that's why I didn't find it.

**Comment by ChrisMcKee** [comment]
@Itay080 no problem; any questions just ask 👍

**Comment by Itay080** [comment]
Hi Chris, I have a minor question which I don't feel like it requires a new issue to be created.
Is the current version from NuGet of Bcrypt.Net using the fixed implementation of Bcrypt? If so, is it safe to use the $2a$ variant?

**Comment by ChrisMcKee** [comment]
@Itay080 
Yes $2a$ is the fixed/valid version for the hash. I know the hash-version explanation is a bit long-winded; but basically in the library default is safe.

The other "versions" were to provide fixes for BSD implementation issues that don't affect this implementation; and the "Y" version was to fix a cockup in springs implementation.

The resulting hash in all instances is the same BAR the version; I altered the code to allow other versions for cross platform compatibility.

tldr: default is safe; the versions correct; unless your taking hashes from some other platform you don't need to worry about the version code 👍
